test/carmen/action_view/helpers/carmen_view_helper_test.rb

```diff
@@ -27,6 +33,10 @@ def test_basic_country_select
     assert_equal_markup(expected, html)
   end
 
+  def document_root_element
+    Nokogiri.parse(@html)
```

Comment:
what's this do? Can't see any uses

Reply:
function `assert_select` wants to see `document_root_element` I don't know how to make document_root_element exactly.
 I made function `document_root_element` and parsed variable @html that existing in each methods :(. If you have an alternative hack, please tell me

Reply:
Fair enough :+1: